### ISSUE TITLE
github pages not working

### ISSUE THREAD

**Issue by Mandeep-009** [vivekchavan14/ShortenURL#1]
the deployed website could not be open. It is showing error 404.

**Comment by Mandeep-009** [comment]
I think you should run all build tasks again in the "actions" section.

**Comment by vivekchavan14** [comment]
Yeah, I had unpublished the site as i had problems on integrating frontend with backend(EJS) , I'll try fixing it again , Thanks for opening the issue.
